src/common/base/test/CMakeLists.txt

```diff
@@ -110,3 +110,11 @@ nebula_link_libraries(
     gtest_main
 )
 nebula_add_test(nebulakey_utils_test)
+
+add_executable(range_vs_transform__bm RangeVsTransformBenchmark.cpp)
+nebula_link_libraries(
+    range_vs_transform__bm
+    follybenchmark
+    boost_regex
```

Comment:
will use `boost_regex` ?

Reply:
because it is depended by folly benchmark